nameko/amqp.py

```diff
@@ -38,12 +39,20 @@ class TestTransport(Transport):
     Connection = ConnectionTester
 
 
-def verify_amqp_uri(amqp_uri):
-    connection = Connection(amqp_uri)
+def verify_amqp_uri(amqp_uri, config=None):
```

Comment:
I like this idea, but still think that `verify_amqp_uri` should either take the whole config object, or the uri and ssl params separately.